### DISCUSSION TITLE
Mute account in Feed (but not groups)

### DISCUSSION THREAD

**Discussion by cobratbq** [jonjomckay/fritter#285] [Ideas]
I have this situation where I follow some accounts that report on service outages. These support accounts typically have less useful replies until I go looking for such status messages. I have made these accounts part of a group dedicated for such messages. It would be helpful  to "silence"/mute them in the main feed (first tab) such that I can make my "primary news feed" more selective and have the ability to look up these accounts through the dedicated group.

Or, alternatively, have the 'Feed' page rely on a 'Feed' group that is pre-existing of which members can be added or removed.

**Comment by ghost** [comment]
This discussion can be closed as feed support has been removed
